inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
I've moved the check to the _MS class' constructor (after checking in w/ @jdevalk if that makes sense in architecture). It will be instanced, but won't run logic unless in multisite context.